src/dragonball/src/api/v1/vmm_action.rs

```diff
@@ -318,6 +318,7 @@ impl VmmService {
             VmmAction::RemoveBlockDevice(drive_id) => {
                 self.remove_block_device(vmm, event_mgr, &drive_id)
             }
+            #[cfg(any(feature = "virtio-net", feature = "vhost-net"))]
```

Comment:
for `UpdateNetworkInterface`, only virtio-net feature is enabled. What if user use `vhost-net` feature?

Reply:
Because vhost-net's rate limiters have been unimplemented so far. See: https://github.com/kata-containers/kata-containers/issues/8327

Reply:
then maybe add a comment near `UpdateNetworkInterface` action?

Reply:
Done.